scalameta/src/main/scala/scala/meta/Origin.scala

```diff
@@ -10,6 +10,8 @@ import org.scalameta.invariants._
   def dialect: Dialect
   def start: Int
   def end: Int
+  def startLine: Int
+  def endLine: Int
```

Comment:
I think it'd be useful to also add these methods to https://github.com/xeno-by/scalameta/blob/master/scalameta/src/main/scala/scala/meta/syntactic/Api.scala#L48.